frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -31,7 +31,7 @@
 
     static final Logger logger = LoggerFactory.getLogger(WorkerThread.class);
 
-    private static final StringDecoder DECODER = new StringDecoder();
+    //private static final StringDecoder DECODER = new StringDecoder();
```

Comment:
This should work, why change it?

Reply:
I got PMD error. So had to use it this way. Is there any issue with this? this should still create one instance of decoder when creating the channel pipeline right?

Reply:
remove "private"

Reply:
Aah. Ok.. But would adding StringDecoder to the channel cause any issue? This is called only when we connect right, so wouldn't this be consistent with other handlers that are registerd??